src/client/routes/Vaults/index.tsx

```diff
@@ -207,21 +211,13 @@ export const Vaults = () => {
   return (
     <ViewContainer>
       <StyledSliderCard
-        header={t('vaults:your-time-card.header')}
+        header="Goblin town or the Citadel?"
```

Comment:
Should this be added to the translation file `public/locales/en/vaults.json`?

Reply:
yes - is it possible to do that as a second task? the socials for this just went live

Reply:
Yes, all good 👍